Tribler/community/multichain/statistics/page_rank.py

```diff
@@ -0,0 +1,114 @@
+"""
+This file contains everything needed to compute page ranks.
+"""
+from random import random, choice
+
+
+class IncrementalPageRank(object):
+    """
+    Class to compute page ranks according to a graph.
+    """
+    def __init__(self, graph, walks_per_node=2, stop_probability=0.1):
+        """
+        Set up the environment by setting a graph, number of walks per node and stop probability.
+        :param graph: the graph for which page ranks have to be computed
+        :param walks_per_node: the number of random walks per node
+        :param stop_probability: the probability to stop in each hop
+        """
+        self.graph = graph
+
+        self.R = walks_per_node
```

Comment:
Maybe we should just rename `self.R` to `self.walks_per_node`, what do you think?